index.html

```diff
@@ -5,11 +5,92 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pencil-ui.netlify.app/css/main.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+        <link rel="stylesheet" href="./style.css">
     <title>eBuy</title>
 </head>
+<style>
+    * {
+        font-family: 'Raleway', sans-serif;
+    }
+</style>
 
 <body>
 
 </body>
+<div class="home_container">
+    <div class="navigation flex shadow">
+        <div class="nav_head">eBuy</div>
+        <div class="nav_input">
+            <input class="input shadow" type="text">
+        </div>
+        <div class="nav_actions flex">
+            <button class="btn secondary_btn">Login</button>
+            <div class="icons">
+                <i class="bi bi-suit-heart icon_sm"></i>
+                <span class="badge badge_small flex flex-center">4</span>
+            </div>
+            <div class="icons">
+                <i class="bi bi-cart icon_sm"></i>
+                <span class="badge badge_small flex flex-center">4</span>
+            </div>
+            <div>Cart</div>
+        </div>
+    </div>
+
+    <img class="responsive_image" src="./assets/music-2694489_1920.jpg" alt="">
+    <div class="homepage_bottom_card flex flex-center">
+        <div class="Horizontal_card flex shadow">
+            <img class="card_image responsive_image" src="./assets/music-4334557.jpg" alt="">
+            <div class="card_detail">
+                <div class="card_name">Apple iPhone 13 128 GB, Starlight</div>
+                <br>
+                <div class="content_about">About the phone</div>
+                <br>
```

Comment:
I am not sure if using `br` tags is considered as good coding practise or not but I would mostly avoid using it very much